### PR TITLE
LPS-105203

### PR DESCRIPTION
**Pull request by erickmonteiroo**

### CODE REVIEW

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/search_columns.jspf [hunk 1]
```diff
@@ -49,11 +49,25 @@
 		/>
 	</c:when>
 	<c:when test='<%= displayStyle.equals("icon") %>'>
+		<liferay-util:buffer var = "customThumbnailHtml">
```
Comment (pei-jung) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/search_columns.jspf:52]: Variable name should be relevant. Maybe rename this to `organizationLogoHtml`.

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/search_columns.jspf [hunk 2]
```diff
@@ -49,11 +49,25 @@
...
+				</div>
+			</div>
+		</liferay-util:buffer>
 		<liferay-ui:search-container-column-text>
-			<liferay-frontend:vertical-card
+			<liferay-frontend:html-vertical-card
 				actionJsp="/organization_action.jsp"
 				actionJspServletContext="<%= application %>"
-				imageUrl="<%= organization.getLogoURL() %>"
+				html="<%= customThumbnailHtml %>"
```
Comment (pei-jung) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/search_columns.jspf:70]: Remember to update the variable here after rename

**Review by pei-jung — CHANGES_REQUESTED** [review]
@erickmonteiroo very well done! 👍  

Everything looks good except one minor variable naming. Please update the code and force push to the branch after you're done.

**Comment by pei-jung** [comment]
@erickmonteiroo make sure to run the source formatter. You can run sf in modules with `gradlew formatsource` or run `ant format-source-current-branch` from the portal-impl directory, which will check only the files that are already commited to current git branch

**Comment by erickmonteiroo** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**